skaffold.yaml

```diff
@@ -9,40 +20,95 @@ build:
   artifacts:
   - image: valory/open-autonomy
     context: deployments/Dockerfiles/autonomy
-    docker:
+    docker: 
       dockerfile: Dockerfile
-      buildArgs:
-        AEA_VERSION: "1.17.0"
-
-
+      buildArgs: 
+        <<: *aea_version
 profiles:
   - name: release
     build:
-        tagPolicy:
-          envTemplate:
-            template: "{{.VERSION}}"
-        artifacts:
-
-        - image: valory/open-autonomy
-          context: deployments/Dockerfiles/autonomy
-          docker:
-            noCache: true
-            dockerfile: Dockerfile
-            buildArgs:
-              AEA_VERSION: "1.17.0"
-
-        - image: valory/open-autonomy-tendermint
-          context: deployments/Dockerfiles/tendermint/
-          docker:
-            noCache: true
-            dockerfile: Dockerfile
-
-        - image: valory/open-autonomy-hardhat
-          context: deployments/Dockerfiles/hardhat
-          docker:
-            noCache: true
-
-        - image: valory/open-autonomy-docs
-          docker:
-            noCache: true
-            dockerfile: deployments/Dockerfiles/documentation/Dockerfile
+      tagPolicy:
+        envTemplate:
+          template: "{{.VERSION}}"
+      artifacts:
+
+      - image: valory/open-autonomy
+        context: deployments/Dockerfiles/autonomy
+        docker:
+          <<: *release_docker_args
+          buildArgs: 
+            <<: *aea_version
+
+      - image: valory/open-autonomy-tendermint
+        context: deployments/Dockerfiles/tendermint/
+        docker:
+          <<: *release_docker_args
+
+      - image: valory/open-autonomy-hardhat
+        context: deployments/Dockerfiles/hardhat
+        docker:
+          <<: *release_docker_args
+
+      - image: valory/open-autonomy-docs
+        docker:
+          <<: *release_docker_args
+          dockerfile: deployments/Dockerfiles/documentation/Dockerfile
+
+      - image: valory/oar-oracle
+        <<: *open_autonomy_runtime_context
+        docker:
+          <<: *release_docker_args
+          buildArgs: &oar-oracle
+            <<: *aea_version
+            AEA_AGENT: "{{.AEA_AGENT_ORACLE}}"
+
+      - image: valory/oar-apy_estimation
+        <<: *open_autonomy_runtime_context
+        docker:
+          <<: *release_docker_args
+          buildArgs: &oar-apy_estimation
+            <<: *aea_version
+            AEA_AGENT: "{{.AEA_AGENT_APY_ESTIMATION}}"
```

Comment:
Why is this different from `latest` below?

Reply:
`AEA_AGENT_APY_ESTIMATION ` is not a version specifier, it's a build arg for oar image.